automerge-frontend/Cargo.toml

```diff
@@ -16,12 +16,17 @@ maplit = "1.0.2"
 thiserror = "1.0.16"
 im-rc = "15.0.0"
 
+[target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dependencies]
```

Comment:
What does this syntax do? I can't find a good description of it anywhere in the rust manifest docs.

Reply:
This sets up some additional dependencies based on the `cfg`. In this case we add some dependencies for working in wasm.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies